os/drivers/input/ist415.c

```diff
@@ -568,6 +573,11 @@ static int ist415_resume_device(struct touchscreen_s *upper)
 		ASSERT(get_errno() == EINTR);
 	}
 
+	if (dev->forcedoff) {
+		sem_post(&dev->sem);
+		return EPERM;
```

Comment:
EPERM is tightly related to permission, so I think ESHUTDOWN or ENXIO is good for this situation, I prefer ESHUTDOWN